account_move_line_search_extension/account.py

```diff
@@ -47,7 +47,8 @@ def search(self, cr, uid, args, offset=0, limit=None, order=None,
         if context and 'account_move_line_search_extension' in context:
             ana_obj = self.pool['account.analytic.account']
             for arg in args:
-                if arg[0] == 'analytic_account_id':
+                if arg[0] == 'analytic_account_id' and\
+                   isinstance(arg[0], basestring):
```

Comment:
that's your personal preference. Our linting rules accept both, so if you insist on this change, please also add a pylint rule enforcing this. My reasoning for doing it this way is that this way, we have the conditions aligned which makes it easier to read imho